packages/admin-client/src/types/langflow-mapping.ts

```diff
@@ -0,0 +1,5 @@
+export interface LangFlowMapping {
```

Comment:
NOTE: In the future we should replace the types with generated types from Swagger